contracts-periphery/Makefile

```
@@ -0,0 +1,39 @@
+# include .env file and export its env vars
+# (-include to ignore error if it does not exist)
+-include .env
+
+all: clean remove install update solc build 
```

After all the other changes, this can become simpler too:
```suggestion
all: clean install test
```